### DISCUSSION TITLE
Upload ignore files/folders with the same name

### DISCUSSION THREAD

**Discussion by mitobin** [rejetto/hfs#235] [Q&A] [answered]
Hi everybody!
When uploading multiple files/folders I want to upload ignore files/folders with the same name already on the sever
Please help
Thank you very much!

**Comment by rejetto** [comment]
if your goal is to preserve files on server, and avoid overwriting, you got the option "Don't overwrite uploading".
This will create a duplicate file on server, tho.
Do you need an option to completely avoid uploading?

**Reply by mitobin** [reply]
i would like to have the option to skip upload when same name. Since that file is already on the server, I don't want to copy it

**Comment by rejetto** [comment]
Do you expect this option to be set by the server administrator, once and for all, or let the user who's uploading decide at the moment of uploading?

**Reply by mitobin** [reply]
let the user who's uploading decide at the moment of uploading

**Reply by mitobin** [reply]
sorry for the slow response because i have to use google translate

**Comment by mitobin** [comment]
Please help 🙏🙏🙏

**Comment by rejetto** [comment]
such feature doesn't exist yet.
I will work on it, but don't have a schedule.
You should receive a notification from this channel when it is ready.

**Reply by mitobin** [reply]
Thanks. Hope to receive a new update from you soon

**Comment by rejetto** [comment, marked as answer]
ok, you got what you asked for in 0.45
you can download beta version at #241
let me know if you like it

<img width="378" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/ee2c239e-553e-4773-a31f-b82d424bc908">

**Reply by mitobin** [reply]
Thanks. the feature works fine. it helps me reduce upload time, no duplication and reduce memory usage
